native-jfr-demo/README.md

```diff
@@ -0,0 +1,72 @@
+# Recording JFR events with Native Image
+
+This example demonstrates how to use the JDK Flight Recorder (JFR) as a tool to collect diagnostic and profiling data in a running Java application, built into a JVM. 
```

Comment:
This example shows how to create a custom JFR event and use that in a native executable. Maybe we could start this guide with an example that shows what standard JFR events are emitted by, for example, `jwebserver`? What do you think?